lib/ansible/parsing/plugin_docs.py

```diff
@@ -26,6 +24,17 @@
 }
 
 
+def _var2string(value):
+    ''' reverse lookup of the dict above '''
+    global string_to_vars
+    string = None
+    for k, v in string_to_vars.items():
+        if v == value:
+            string = k
```

Comment:
I don't really love this function, but since it's an "end of the road" call, I suppose it doesn't matter much.  Seems like you could just `return` here instead of setting a var and doing a `break`.  And potentially have the function raise an exception at the end otherwise.

Reply:
initial versino had `else: raise keyerror`  but i really didn't see this useful/possible since at this point